.github/workflows/grid_client_nightly.yml

```diff
@@ -45,56 +39,62 @@ jobs:
       - name: Install deps
         run: |
           sudo apt-get update
-          sudo apt-get install -y git libtool tmux redis net-tools
+          sudo apt-get install -y git libtool tmux redis net-tools gh jq
```

Comment:
AFAK, JQ is a JSON formatted, why could we use such a package in a yaml workflow?

```sh
echo '{ "name":"John", "age":31, "city":"New York" }' | jq .
```

Reply:
Removed.